### PR TITLE
Fix node up

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
issue: #22782

### CODE REVIEW

File: internal/querycoordv2/server.go [hunk 1]
```diff
@@ -646,6 +650,44 @@ func (s *Server) watchNodes(revision int64) {
...
+		select {
+		case <-s.ctx.Done():
+			return
+
+		case <-ticker.C:
+			ctx, cancel := context.WithTimeout(s.ctx, Params.QueryCoordCfg.CheckHealthRPCTimeout.GetAsDuration(time.Millisecond))
+			defer cancel()
+			reasons, err := s.checkNodeHealth(ctx)
+			if err != nil {
+				log.RatedWarn(10, "unhealthy node exist, node up will be delayed",
```
Comment (jiaoew1991) [inline internal/querycoordv2/server.go:669]: use WithRateGroup？

Comment (weiliu1031) [reply internal/querycoordv2/server.go:669]: WithRateGroup Has been specified above

**Pull request merged**